### ISSUE TITLE
Mapping core vocabularies

### ISSUE THREAD

**Issue by muricna** [OP-TED/ted-rdf-mapping#218]
Although the mappings in section 1 point to cpov:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/muricna/AppData/Local/Temp/1/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/muricna/AppData/Local/Temp/1/msohtmlclip1/01/clip_filelist.xml">

<style>

</style>
</head>

<body link="#1155CC" vlink="#1155CC">

I.1.8 | Contact person | epo:AgentInRole (from   CL1) / cpov:ContactPoint / rdf:langString | ?this epo:hasContactPointInRole /   epo:hasContactName ?value   .
-- | -- | -- | --
I.1.9 | Telephone | epo:AgentInRole (from CL1) / cpov:ContactPoint /   rdf:langString | ?this epo:hasContactPointInRole  / cpov:telephone ?value .
I.1.10 | E-mail | epo:AgentInRole (from CL1) / cpov:ContactPoint /   xsd:anyURI | ?this epo:hasContactPointInRole  / cpov:email ?value .
I.1.11 | Fax | epo:AgentInRole (from CL1) / cpov:ContactPoint /   rdf:langString | ?this epo:hasContactPointInRole  / epo:hasFax ?value .
I.1.12 | Internet Addresses |   |  
I.1.12.1 | Main address | epo:AgentInRole (from   CL1) / cpov:ContactPoint / xsd:anyURI | ?this epo:hasContactPointInRole /   epo:hasInternetAddress ?value .
I.1.12.2 | Address of the buyer   profile | epo:AgentInRole (from CL1) / epo:BuyerProfile /   xsd:anyURI | ?this epo:hasBuyerProfile /   epo:hasURL ?value .

</body>

</html>

**Comment by csnyulas** [comment]
Indeed, this is due to the fact that both the `cpov:`, `cccev:` and `cv:` prefixes (and some others) point to the same namespace. In cases when properties/classes with multiple of these prefixes are used in the mappings the RMLMapper uses only one prefix declarations in the generated Turtle output. It is not clear, based on what logic is the used prefix chosen. We have not found a way, so far, to control which prefix to be used in the output, or to make it possible to generate multiple of these prefixes that stand for the same namespace in the output.

Technically, this should not matter, really, because those prefixes are just abbreviations, to aid human consumptions, and any triplestore that would load this data would resolve it to the full IRI anyways. It is strange only for the human, who looks at it, and expects a certain prefix, and sees a different one.

A possible solution would be to use only one prefix in EPO for all the Core Vocabularies that are in the http://data.europa.eu/m8g namespace. If using one of the official ones would be inappropriate, or confusing, we could use a more neutral one, such as `cv:` or `core-voc:` for all properties/classes that are reused from them in EPO.

BTW, looking at the latest versions of the core vocabularies, even a random scan revealed some apparent problems with some of the reused terms. For example, just considering the terms reused from CCCEV in the context of our mapping, we found in the conceptual mapping three different properties: `cccev:name`, `cccev:description` and `cccev:weight`. Out of these only the `cccev:weight` is listed in the official RDF version of version to of CCCEV: https://semiceu.github.io/CCCEV/releases/2.00/voc/cccev.ttl . Where are `cccev:name` and `cccev:description` coming from?

@andreea-pasare We should probably open an EPO ticket to verify **all** reused terms, and perhaps unify the prefixes of terms coming from the same namespace, if it makes sense and can be done. I also wanted to mention here the `legal:registeredAddress` property, which I already flagged some time ago, but it seem like that has been dealt with in this issue: https://github.com/OP-TED/ePO/issues/384
